resources/en-US/training/t-z.training.bxb

```diff
@@ -1,10 +1,20 @@
+train (t-z285r9g9qcqbpz6zl3e4xkhhi) {
+  utterance ("[g:CommitOrder:continue,r:UpdateOrder] Remove the {[g:RemovedItem] (polo)[v:SearchTerm]} and the {[g:RemovedItem2] (v-neck)[v:SearchTerm]}")
```

Comment:
nitpick - the model name should be `ItemToRemove`, since it hasn't happened yet. `RemovedItem` implies it's already gone, like i'm talking about the item i removed previously.

Reply:
Valid point. This will be a separate task though.